### ISSUE TITLE
Images are not showing in UWP App and Android APK crash

### ISSUE THREAD

**Issue by hanatharesh2712** [ilia-valchenko/XamarinEpubReader#6]
Hello @ilia-valchenko , Thanks for making this. 

I am using EpubReader library to read content from the epub file and display in the APP. but, my epub file contains the images and those images are not displays when I see in the UWP APP.

also, When I build Android APK to install in my device. the build was succeeded but when I open the app in my device it crashes. 

Can you help me to resolve these issues?

Thanks in advance!

**Comment by ilia-valchenko** [comment]
@hanatharesh2712 Yep. This functionality has not implemented yet. The problem is that I grab all book pages into single html page. The paths of images become invalid.

**Comment by aamir-munir** [comment]
I tried running it in Visual Studio but got error.

**Comment by aamir-munir** [comment]
this is the error I am receiving:

'DroidFiler' does not implement interface member 'IFiler.DoesFileExistAsync(string)'

four other errors like that.
